1.wave2_data/3.processes_features/extract_image_features_interstellar.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Process image features from CellProfiler readout
+
+# ## Import Libraries
+
+# In[1]:
+
+
+import pathlib
+import pandas as pd
+import numpy as np
+
+from pycytominer.cyto_utils import cells, output, util
+
+import sys
+sys.path.append("../../utils")
+import extract_image_features_utils as extract_utils
+
+
+# ## Set up paths to CellProfiler directory and outputs
+
+# In[2]:
+
+
+# Set file and directory constants
+cp_dir = "../2.cellprofiler_analysis"
+output_dir = "data"
+
+
+# ## Set paths to sqlite files
+
+# In[3]:
+
+
+# Set name and path of .sqlite file and path to metadata
+sql_file = "interstellar_wave2.sqlite"
+single_cell_file = f"sqlite:///{cp_dir}/analysis_output/{sql_file}"
+platemap_file = "../../0.wave1_data/2.cellprofiler_analysis/metadata/Interstellar_wave1_platemap.csv"
```

Comment:
It is the same platemap as wave 1? If so, please add a comment to make sure that folks understand (potentially confusing!)

Reply:
This will be corrected in the next PR. I will make a `metadata` folder like the `utils` folder since all the waves will be using this platemap once corrected. I will also change the name to avoid confusion.